docs/install/project.md

```diff
@@ -11,10 +11,10 @@ terminal.
 * Then go to a folder where you want to create a project in a subfolder
   `my_project`, and type:
 	```bash
-	leanpkg new my_project
+	leanpkg +leanprover-community/lean:3.6.1 new my_project
```

Comment:
This is outdated. The new workflow is `leanproject new my_project`, nothing else.

Reply:
Sweet.